src/applications/representative/components/search/SearchControls.jsx

```diff
@@ -183,29 +147,21 @@ const SearchControls = props => {
             onChange={onChange}
           />
 
-          <div>
-            <div>
-              <label
-                htmlFor="representative-organization"
-                id="representative-organization-label"
-              >
-                {representativeType === 'Veteran Service Organization (VSO)'
-                  ? 'Organization'
-                  : 'Representative'}{' '}
-                name{' '}
-              </label>
-            </div>
-            <input
-              id="representative-organization"
-              ref={repInputFieldRef}
-              name="representative-organization"
-              type="text"
-              onChange={handleRepOrganizationChange}
-              onBlur={handleRepOrganizationBlur}
-              value={repOrganizationInputString}
-              title="Organization or Representative Name"
-            />
-          </div>
+          <va-text-input
+            hint={null}
+            label={
+              representativeType === 'Veteran Service Organization (VSO)'
+                ? 'Organization Name'
+                : 'Representative Name'
+            }
+            message-aria-describedby="Text input for organization or representative name"
+            name="Organization or Representative Name"
+            onChange={handleRepOrganizationChange}
+            // onBlur={handleRepOrganizationBlur}
```

Comment:
Was this intentionally left in?

Reply:
Good call, unintentional but I may be re-introducing it (will address in my next PR).